spanner/transaction.go

```diff
@@ -874,6 +922,10 @@ func (t *ReadWriteTransaction) BatchUpdate(ctx context.Context, stmts []Statemen
 		Transaction: ts,
 		Statements:  sppbStmts,
 		Seqno:       atomic.AddInt64(&t.sequenceNumber, 1),
+		RequestOptions: &sppb.RequestOptions{
+			RequestTag:     opts.RequestTag,
+			TransactionTag: t.qo.transactionTag,
```

Comment:
What I mean is that we extract `TransactionTag` from `t.to` (transaction options) instead of `t.qo` (query options)? Just a thought.

Reply:
Good point, done (see also above).